css/style.css

```diff
@@ -0,0 +1,442 @@
+
+
+.w-container{
+	max-width: 1300px;
+	margin:auto;
+	background: #fff;
+	margin-bottom: 2rem;
+	padding-bottom: 2rem;
+}
+
+.w-container .navbar-nav{
+	flex:1;
+}
+
+
+.w-container .navbar-nav .nav-item a{
+	font-weight: 700;
+	color:#000;
+	font-size: 1.25rem;
+	padding:0rem 1rem;
+	border-right: 1px solid #e5e5e5;
+}
+
+.logo-area{
+	background: #f72210;
+}
+
+.logo-img{
+	position: relative;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.